CMakeLists.txt

```diff
@@ -249,6 +261,13 @@ if(NOT IREE_CUDA_AVAILABLE OR CMAKE_CROSSCOMPILING)
   set(IREE_HAL_DRIVER_CUDA_DEFAULT OFF)
 endif()
 
+# HIP support is enabled by default if not cross compiling. Note: a HIP-compatible
```

Comment:
I guess we should also update https://iree.dev/guides/deployment-configurations/gpu-rocm/ (somehow)

Reply:
Yup. I'll do that next, once we have python packages including it. (probably need to update the doc page in some major ways too.)